tests/OmniSharp.Roslyn.CSharp.Tests/HighlightFacts.cs

```diff
@@ -7,178 +7,146 @@
 using Xunit;
 using Xunit.Abstractions;
 
-namespace OmniSharp.Roslyn.CSharp.Tests
-{
-    public class HighlightFacts : AbstractSingleRequestHandlerTestFixture<HighlightingService>
-    {
-        public HighlightFacts(ITestOutputHelper output, SharedOmniSharpHostFixture sharedOmniSharpHostFixture)
-            : base(output, sharedOmniSharpHostFixture)
-        {
-        }
-
-        protected override string EndpointName => OmniSharpEndpoints.Highlight;
-
-        [Fact]
-        public async Task HighlightSingleLine()
-        {
-            var testFile = new TestFile("a.cs", @"
+namespace OmniSharp.Roslyn.CSharp.Tests {
```

Comment:
It seems the formatting in this file is a bit broken. The braces should be on the next line